metadata_fetcher/fetchers/Fetcher.py

```diff
@@ -60,15 +60,20 @@ def fetch_page(self):
                 f"[{self.collection_id}]: unable to fetch page {page}")
 
         record_count = self.check_page(response)
+        if not record_count:
+            logger.warning(
+                f"[{self.collection_id}]: no records found "
+                f"on page {self.write_page}"
+            )
```

Comment:
Does it makes sense to raise an error instead? 

`raise ValueError(f"[{self.collection_id}]: no records found on page {self.write_page} at {response.request.url}")`

Reply:
Also, I don't see the logger messages in Airflow, do you??? I only see `print` messages.

Reply:
Yes, I see logger messages here. I only couldn't get logger messages to appear in `metadata_mapper/validate_mapping.py`, which is why that's a print statement.